documentation/src/docs/pravega-concepts.md

```diff
@@ -376,64 +269,54 @@ The following figure depicts the components deployed by Pravega:
-storage is comprised of two tiers: Tier 1 Storage, which provides short term,
-low-latency, data storage,  guaranteeing the durability of data written to
-Streams and Tier 2 Storage providing longer term storage of Stream data.
- Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
-Tier 1 Storage and uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud
-Storage (ECS) to implement Tier 2 Storage.  Tier 1 Storage typically runs within
-the Pravega cluster.  Tier 2 Storage is normally deployed outside the Pravega
-cluster.
-
-Tiering storage is important to deliver the combination of fast access to Stream
-data but also allow Streams to store a vast amount of data.  Tier 1 storage
-persists the most recently written Stream data.  As data in Tier 1 Storage ages,
+coordinated to run Pravega called a **Pravega cluster**.  
+
+Pravega presents a software-defined storage (SDS) architecture formed by **Controller** instances
+(_control plane_) and Pravega Servers (_data plane_).The set of Pravega Servers is collectively known as the **Segment Store**. 
+
+The set of Controller instances together forms the control plane of Pravega, providing
+functionality to _create, update_ and _delete_ Streams. Further, it extends the functionality to retrieve information about the Streams, monitor the health of the Pravega cluster, gather metrics, etc.  There
+are usually multiple (recommended at least 3) Controller instances running in a running in a cluster for high availability.  
+
+The [Segment Store](Segment-store-service.md) implements the Pravega data plane.
+Pravega Servers provide the API to read and write data in Streams. Data storage is comprised of two tiers:
+- **Tier 1:** It provides short term, low-latency, data storage, guaranteeing the durability of data written to Streams. Pravega uses [Apache Bookkeeper](http://bookkeeper.apache.org/) to implement
+Tier 1 Storage.
+- **Tier 2:** It provides long term storage for Stream data. Pravega uses HDFS, Dell EMC's Isilon or Dell EMC's Elastic Cloud Storage (ECS) to implement Tier 2 Storage.
+
+Tier 1 Storage typically runs _within_ the Pravega cluster. Tier 2 Storage is normally deployed _outside_ the Pravega cluster. Tiering storage is important to deliver the combination of fast access to Stream
+data and also allows large data storage for Streams. Tier 1 storage
+persists on the most recently written Stream data. As data in Tier 1 Storage ages,
 it is moved into Tier 2 Storage.
 
 Pravega uses [Apache Zookeeper](https://zookeeper.apache.org/) as the
 coordination mechanism for the components in the Pravega cluster.  
 
-Pravega is built as a data storage primitive first and foremost.  Pravega is
-carefully designed to take advantage of software defined storage so that the
+Pravega is built as a data storage primitive first and foremost. Pravega is
+carefully designed to take advantage of software-defined storage, so that, the
```

Comment:
Perhaps the second comma is unnecessary: `so that(,)`

Reply:
done